### ISSUE TITLE
Escrow Transactions are not Registering on my End (Vendor Side)

### ISSUE THREAD

**Issue by cryptoguard** [particl/particl-desktop#1321]
Trying to accept orders for the lapel pins. Customers put their funds in escrow, but my still says Awaiting Buyer: 

Here is an example with Carmen's order. On her side, it shows this (she has locked her funds in escrow): 
![escrow](https://user-images.githubusercontent.com/27031337/50009170-4c8ee800-ff84-11e8-8e08-dc3834936832.jpg)

On my end, it still says that I am waiting for her to put her funds in escrow: 

![awaiting](https://user-images.githubusercontent.com/27031337/50009208-5c0e3100-ff84-11e8-8660-7282a9a9e6c0.jpg)

**Comment by zaSmilingIdiot** [comment]
This error occurred as a result of the smsg message processor in the marketplace not being able to parse and save certain smsg messages (mostly due to DB write locking issues). In which case the processor would indicate to the core that the message could be deleted, as if it was a successfully processed message.

At least as of marketplace v0.0.53, this has be resolved: various optimizations and changes have been applied; for example: making use of batched DB writes, updated parsing, not deleting un-processed messages, etc. Further optimizations have also been applied in the next version of the marketplace, which should clean up any latent issues tracing their way back to this particular error. 

Simply put, this should now be resolved.
